homeassistant/components/incomfort/climate.py

```diff
@@ -68,6 +68,14 @@ def __init__(
         self._legacy_setpoint_status = legacy_setpoint_status
 
         self._attr_unique_id = f"{heater.serial_no}_{room.room_no}"
+        if coordinator.unique_id:
+            self._attr_device_info = DeviceInfo(
+                identifiers={(DOMAIN, self._attr_unique_id)},
+                manufacturer="Intergas",
+                name=f"Thermostat {room.room_no}",
+                via_device=(DOMAIN, coordinator.unique_id),
+            )
+            return
```

Comment:
You can do `self._attr_device_info["via_device"] = ...` instead to avoid having this if statement and having 2 DeviceInfo definitions

Reply:
Ah, it is a `dict`, of course :-)